housekeeper/cli/delete.py

```diff
@@ -99,54 +101,79 @@ def version_cmd(context, bundle_name, version_id, yes):
 @click.pass_context
 def files_cmd(context, yes, tag, bundle_name, before, notondisk, list_files, list_files_verbose):
```

Comment:
Maybe add type hints here

Reply:
This function has a looot of arguments 😟 Maybe consider fixing this?

Reply:
The function is a click command and the parameters are the different arguments the command accepts, so I'm not sure if it is possible without changing the CLI. I think it is out of scope for this PR

Reply:
fair enough